packages/volto/src/components/manage/Form/Field.jsx

```diff
@@ -172,6 +174,19 @@ const UnconnectedField = (props, { intl }) => {
     return null;
   }
 
+  if (props.mode === MODE_DISPLAY) {
+    let field = {
+      ...props,
+      //widget: getWidget(f, contentSchema?.properties[f]), //import { getWidget } from '@plone/volto/helpers/Widget/utils';
+    };
+    const DisplayWidget = config.widgets?.views?.getWidget(field);
```

Comment:
We should make this consistent with how DisplayView gets the widget (https://github.com/plone/volto/blob/main/packages/volto/src/components/theme/View/DefaultView.jsx#L81)